meetings/tool/src/main/resources/create-meeting.properties

```diff
@@ -5,6 +5,7 @@ cancel=Cancel
 close_date=Close Date
 error_create_meeting_500=There was a problem creating the meeting
 error_create_meeting_unknown=Unknown error when trying to create meeting. Response code:
+error_video_conferencing_config=There is no configuration of the video conferencing service. Please check this with your Sakai administrator.
```

Comment:
The meetings tool requires at least one provider like Microsoft Teams or Zoom, please contact your service administrator or vendor.